index.html

```diff
@@ -3761,6 +3767,66 @@ <h3><dfn>Get Window Handles</dfn></h3>
 </aside>
 </section> <!-- /Get Window Handles -->
 
+<section>
+<h3><dfn>Create Window</dfn></h3>
+
+<table class="simple jsoncommand">
+ <tr>
+  <th>HTTP Method</th>
+  <th>URI Template</th>
+ </tr>
+ <tr>
+  <td>GET</td>
+  <td>/session/{<var>session id</var>}/window/new</td>
+ </tr>
+</table>
+
+<p>Create a new top level browsing context.
+
+<p>The <a>remote end steps</a> are:
+
+<ol>
+ <li><p>If there is an active <a>user prompt</a>, that prevents the
+  creation of another <a>top-level browsing context</a>,
+  return <a>error</a> with <a>error code</a> <a>unexpected alert open</a>.
+
+ <li><p>If the implementation does not support creating new top-level
+  browsing contexts, return <a>error</a> with <a>error code</a>
+  <a>unsupported operation</a>.
+
+ <li><p>Let <var>type hint</var> be the result of <a>getting the property</a>
+  "<code>type</code>" from the <var>parameters</var> argument.
+
+ <li><p>Create a new <a>top-level browsing context</a>. If <var>type
+  hint</var> has the value "<code>tab</code>" and the implementation
+  supports multiple browsing context in the same OS window, the new
+  browsing context should share an OS window with the <a>current
+  browsing context</a>, otherwise the created browsing context should be in a
+  new OS window.
+
+ <li><p>Let <var>handle</var> be the
+  associated <a>window handle</a> of the newly created window.
+
+ <li><p>Let <var>type</var> be "<code>tab</code>" if the newly created
+  window shares an OS-level window with the <a>current browsing
+  context</a>, or "<code>window</code>" otherwise.
```

Comment:
We talked about letting remote ends return what makes sense to them, but to _suggest_ returning `tab` or `window`.

Reply:
also should this define an error to be thrown if `type` is neither `tab` nor `window`

Reply:
Not throwing an error was intentional so that we can extend the list in the future without breaking compat. (basically new values will always look like `window` in old drivers).

Reply:
@jgraham in case of selector strategies we throw an `Invalid Selector` error, shouldn't we stay consistent in that sense?

Reply:
@christian-bromann We discussed the need to meet vendor’s future requirements for window types that is something other than `tab` or `window`, e.g. `hololens`, `space`, or `halfspace`. We also talked about how making this infallible is in line with existing commands.

We want to provide _a hint_ to the browser describing what the most desirable type of new top-level browsing context would be, to optimise for scripts that work with minimal product-specific knowledge cross-browser. If the browser can’t open a new top-level browsing context, it obviously makes sense to error with an unsupported operation.

Reply:
I think it makes more sense to require remote ends to return specific things at the moment; we probably don't want an unconstrained extension space here.